asmcomp/cmm.mli

```diff
@@ -134,10 +134,10 @@ and expression =
   | Clet of Ident.t * expression * expression
   | Cassign of Ident.t * expression
   | Ctuple of expression list
-  | Cop of operation * expression list
+  | Cop of operation * expression list * Debuginfo.t
   | Csequence of expression * expression
   | Cifthenelse of expression * expression * expression
-  | Cswitch of expression * int array * expression array
+  | Cswitch of expression * int array * expression array * Debuginfo.t
```

Comment:
I'm a bit torn on those ones. In a sense `mov` and `jump` don't really do anything and so never really map back properly to the source program. The registers and the code layout bare only a passing resemblance to the original code, so the user doesn't care about them. Essentially, I see these instructions as just artifacts of the need to use a finite set of registers and a linear code representation, so for my reasoning about what gets some debug info I think they in some sense "don't count".